src/sections/Community/Newcomers-guide/index.js

```diff
@@ -76,8 +78,8 @@ const NewcomersGuide = () => {
         </div>
         <div className="instructions">
           <Row className="step">
-            <Col sm={12} lg={6}>
-              <img src={CommunityImage1} className="finding-an-issue" alt="Community Image" />
+            <Col sm={12} lg={6} className="finding-an-issue" >
+              <StaticImage src={CommunityImage1} alt="Community Image" />
```

Comment:
This one is getting messed up. You can let this be a normal `<img>` ig since it's an svg. Rest looks good.

Reply:
ok, sure